packages/transformers/sass/src/SassTransformer.js

```diff
@@ -40,7 +40,11 @@ export default new Transformer({
       config = {};
     }
 
-    config.file = asset.filePath;
+    if (asset.isInline) {
```

Comment:
Not sure I fully understand this change. Could you elaborate?

Reply:
To compile sass we cannot give it a file because the inline data is not a file, we just have the content to conform with their API, we need to pass it as data is we have a string of content instead of file. 

The indent further down could be done better but is based on the type and that we don't have a config for inline content. The expectation is that they are setting the type of the style to sass because they want indent syntax instead of scss with block syntax.

Reply:
Huh, didn't realize we weren't passing the code in all the time. Seems like it would be problematic for even normal assets for SASS to be responsible for reading the file because it won't use our virtual FS. Can revisit that later.